docs/features/Community Questions & Stories.md

```
@@ -609,17 +608,18 @@ Staff fills out question form on Manage > Questions tab
       -> Flux::toast('Question created!', variant: 'success')
 ```
 
-### Automatic Question Lifecycle (Scheduled)
+### Automatic Question Lifecycle
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)